### PR TITLE
add release notes

### PR DESCRIPTION
**Pull request by laura-ham**
This change is [Reviewable](https://reviewable.io/reviews/Chainlit/literal-docs/71)

### CODE REVIEW

File: more/release-notes.mdx [hunk 1]
```diff
@@ -4,6 +4,44 @@ title: Release Notes
...
+* [TypeScript SDK](/typescript-client) version `0.0.503` and above.
+
+### Breaking changes
+
+None
+
+### New features
+
+* We've adapted the Online Evaluation page for our on-premise clients who may not have a Queuing system installed
+* Interactions with filters and layouts on the dashboard are now tracked on PostHog
```
Comment (Dam-Buty) [inline more/release-notes.mdx:21]: i don't know if i'd mention that to end users ? it's mostly to our advantage and doesn't really bring them any benefit

File: more/release-notes.mdx [hunk 2]
```diff
@@ -4,6 +4,44 @@ title: Release Notes
...
+
+* We're continuing our push towards a more consistent - and prettier - User Experience:
+  * We've switched to a more vibrant color scheme
+  * Made some visual tweaks on the Dashboard page
+  * Observability items such as Threads, Runs, and others will now display as full pages rather than side-panes
+  * And lots of other improvements across the platform
+* Some changes on the way the platform is deployed, both on our end and for our on-premise users :
+  * Improved and centralized environment management
+  * The Portkey AI Gateway is now handled directly inside the Node process
+  * The `BUCKET_NAME` environment variable is no longer variable. Trying to store objects will log errors but not disrupt the rest of the operations
```
Comment (Dam-Buty) [inline more/release-notes.mdx:34]: ooops my mistake :see_no_evil: 

```suggestion
  * The `BUCKET_NAME` environment variable is no longer mandatory. Trying to store objects will log errors but not disrupt the rest of the operations
```

**Review by Dam-Buty — CHANGES_REQUESTED** [review]
just 2 minor comments

File: more/release-notes.mdx (hunk 1 above)
Comment (laura-ham) [reply more/release-notes.mdx:21]: Fair point. I thought it would be useful for on-premise users, but I'll remove it.

Comment (Dam-Buty) [reply more/release-notes.mdx]: I see ! This isn't a worry as Posthog tracking isn't active for our on-premise users.

**Review by Dam-Buty — APPROVED** [review]
Good for me !

**Pull request merged**